### PR TITLE
Format the new/edit/reply forms for forum posts

### LINKED ISSUE #1316: Forum new post layout
Topic field is very short compared to the body, and the label for Body is at the bottom of the field. I think it'd look better at the top or above the field
Screen Shot 2019-05-14 at 7 09 04 PM

### PR DESCRIPTION
**Pull request by alastair**
**Issue(s)**
Fixes #1316

**Description**
Quick fix to make the new thread page a bit nicer looking. Small fix, so I decided to do it now instead of waiting 12 more months :)

### CODE REVIEW

File: forum/forms.py [hunk 1]
```diff
@@ -19,12 +19,16 @@
...
 from django import forms
+
 from utils.forms import HtmlCleaningCharField
 from utils.spam import is_spam
 
+
 class PostReplyForm(forms.Form):
-    body = HtmlCleaningCharField(widget=forms.Textarea(attrs=dict(cols=100, rows=20)))
-    subscribe = forms.BooleanField(help_text="Send me an email notification when new posts are added in this thread.", required=False, initial=True)
+    body = HtmlCleaningCharField(widget=forms.Textarea(attrs={'cols': 100, 'rows': 30}))
```
Comment (alastair) [inline forum/forms.py]: changed from 20 to 30 to make it the same as new thread. ok?

File: templates/forum/new_thread.html [hunk 2]
```diff
@@ -13,10 +13,22 @@
...
 
-
-<table>
-{{form.as_p}}
-</table>
+<div>
+    {{ form.non_field_errors }}
+    {{ form.title.errors }}
+<p>
+    <label for="{{ form.title.id_for_label }}">Thread title:</label><br>
```
Comment (alastair) [inline templates/forum/new_thread.html]: could have been done as `{{ form.title.label_tag }}`, but I wanted to change the label text from 'title' to 'thread title'

File: templates/forum/post_edit.html [hunk 3]
```diff
@@ -12,15 +12,14 @@ <h2>Edit post</h2>
...
-        <br>
-        {{ form.errors }}
-        <br>
-        {% for field in form %}
-            {% if field.label == 'Body' %}
-                {{ field }}
-            {% endif %}
-        {% endfor %}
-        <br><br>
+<div>
```
Comment (ffont) [inline templates/forum/post_edit.html]: heh, much better now

File: templates/forum/reply.html [hunk 4]
```diff
@@ -12,7 +12,17 @@
 
 <form method="post" action="." class="disable-on-submit">{% csrf_token %}
 {% include 'forum/html_explanation.html' %}
-{{form.as_p}}
+<div>
+    {{ form.non_field_errors }}
+    {{ form.body.errors }}
+<p>
+    <label for="{{ form.body.id_for_label }}">Body:</label><br>
```
Comment (alastair) [inline templates/forum/reply.html]: What do you think about calling this "post" or "message" instead of body?

Comment (ffont) [reply templates/forum/reply.html]: Done in https://github.com/MTG/freesound/pull/1318/commits/59ff0d2edd83899af58c20b50b9ab4250c5bbb0c

**Pull request merged**